zerver/openapi/zulip.yaml

```diff
@@ -11931,6 +11933,16 @@ paths:
                   items:
                     type: integer
                   example: [1, 2]
+                notify_referrer_on_join:
+                  description: |
+                    A boolean indicating whether the referrer has chosen to receive direct notifications
+                    from the notification bot. If `false` the referrer will not receive a notification for
+                    accepted invitations.
+
+                    **Changes**: New in Zulip 9.0 (feature level 247), added option for referrer to
+                    receive direct notifications from the notification bot when an invited user joins.
```

Comment:
No need to explain the use of this field here again. It is sufficient to just write "New in Zulip 9.0 (feature level 247)."